app/controllers/alchemy/json_api/pages_controller.rb

```diff
@@ -50,7 +50,7 @@ def page_scope
       def page_scope_with_includes
         base_page_scope.
           with_language(Language.current).
-          preload(language: { nodes: [:parent, :page] }, all_elements: { contents: { essence: :ingredient_association } })
+          preload(language: { nodes: [{ parent: :page}, :page, { children: :page }] }, all_elements: { contents: { essence: :ingredient_association } })
```

Comment:
I don't think loading the associated pages of the parent and the children is necessary, as loading them with the nodes should be enough. This is loading more data than we need.

Reply:
I did not made any difference. The amount of Page queris stays the same. Rails seems to be pretty smart in not eager loading relations more than needed. But still we should remove the etxtra code since it is not necessary as you already mentioned. Thanks!